driver/driver.go

```diff
@@ -28,8 +28,8 @@ type driverFlags struct {
 	targetPatternFile string
 	revisionBefore    string
 	manualTestMode    string
-	// Expected option: "build"
-	bazelCommandOverride string
+	// Expected value: "build"
+	bazelSubCommandOverride string
```

Comment:
Do you have ambition have this option do more than you're doing with it here? If not, you could name it something like "forceUseOfBuildForTests" and treat it as a boolean option, defaulting to false.

As named, it sounds like you're encouraging callers to supply a command name, but then you don't wind up actually using that command name unless it's "build".

Reply:
Updated, thank you. I did start with the idea to allow the user to set either build/test, but chose to just force a build (I couldn't think of examples where one might want to force a test).